src/services/safari-view-controller.ts

```diff
@@ -0,0 +1,29 @@
+import { SafariViewController } from '@ionic-native/safari-view-controller'
+import { InAppBrowser } from '@ionic-native/in-app-browser'
+
+const openWithNativeBrowser = async (url: string) => {
+    InAppBrowser.create(url, '_system')
+}
+
+export const openWithSafariViewController = async (url: string) => {
+    const isAvailable = await SafariViewController.isAvailable()
+    if (isAvailable) {
+        SafariViewController.show({
+            url,
+            hidden: false,
+            animated: false,
+            transition: 'curl',
+            enterReaderModeIfAvailable: true,
+            tintColor: '#ff0000',
+        }).subscribe(
+            (result: any) => {
+                if (result.event === 'opened') console.log('Opened')
```

Comment:
is it necessary ? May be we should do this `console.log(result.event)`

Reply:
we don't actually use that indeed, at least for the moment. It comes from the example code